app/assets/config/manifest.js

```diff
@@ -1,2 +1,2 @@
-//= link_tree ../images
-//= link_directory ../stylesheets .css
+// = link_tree ../images
+// = link_directory ../stylesheets .css
```

Comment:
we are still currently using rails built in sass compiler but not sure about how this bit works. the js stuff gets its own manifest created by webpacker.